src/Sulu/Bundle/MediaBundle/Content/Types/MediaSelectionContentType.php

```diff
@@ -70,7 +70,7 @@ public function read(
     ) {
         $data = json_decode($node->getPropertyValueWithDefault($property->getName(), '{"ids": []}'), true);
 
-        $property->setValue(isset($data['ids']) ? $data : null);
+        $property->setValue(isset($data['ids']) ? [] : null);
```

Comment:
???

Reply:
@nnatter Think you accidently pushed this

Reply:
🙈

Reply:
yes, this is the code for testing the change - sorry 😇